metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/UdpTransport.java

```diff
@@ -22,6 +24,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UdpTransport.class);
   private final StatsDClient statsd;
+  private final HashMap lastSeenCounters = new HashMap<String,Integer>();
```

Comment:
re: 1, where is the timgroup statsd client being included? I don't see it in the poms and it's arguing with me that it won't work with a long, [which it clearly does](https://github.com/tim-group/java-statsd-client/blob/master/src/main/java/com/timgroup/statsd/StatsDClient.java#L37)…